include/toml/toml.h

```diff
@@ -25,6 +19,12 @@
 #include <utility>
 #include <vector>
 
+#if defined(_MSC_VER)
+#undef sscanf
+//NOTE: sscanf_s requires to specify sizes of buffers to store %c or %s
+#define sscanf sscanf_s
```

Comment:
Yeah, you definitely right about it now i think about it.
I'll remove ifdef compeltely i think and just move usage of correct function on line 894